critiquebrainz/frontend/external/musicbrainz_db/recording.py

```diff
@@ -0,0 +1,22 @@
+from brainzutils import cache
+from brainzutils.musicbrainz_db.recording import fetch_multiple_recordings
+from critiquebrainz.frontend.external.musicbrainz_db import DEFAULT_CACHE_EXPIRATION
+
+
+def get_recording_by_id(mbid):
+    """Get recording with MusicBrainz ID.
+
+    Args:
+        mbid (uuid): MBID(gid) of the recording.
+    Returns:
+        Dictionary containing the recording information
+    """
+    key = cache.gen_key(mbid)
+    recording = cache.get(key)
+    if not recording:
+        recording = fetch_multiple_recordings(
```

Comment:
any particular reason why we use the fetch_multiple method instead of the already existing `get_recording_by_mbid` in BU?

Reply:
I think these functions got moved to BU but this code was not updated.

Reply:
Fixed.